BUILDING_ON_FREEBSD.md

```diff
@@ -10,6 +10,7 @@ FreeBSD 13.0-CURRENT.
 
 1. Install build dependencies from package sources (or build from the
    ports tree): `# pkg install qt5-core qt5-multimedia qt5-svg qt5-qmake qt5-buildtools gstreamer-plugins-good boost-libs rapidjson`
-1. go into project directory
-1. create build folder `$ mkdir build && cd build`
-1. `$ qmake .. && make`
+1. Go into the project directory
+1. Create a build folder and open it (`mkdir build && cd build`)
```

Comment:
If you want to be very specific, you don't open a directory with `cd` command, but rather "go to it". Opening a directory would mean opening it in some file explorer or anything. Original line is clear enough imo and is not worth changing

Reply:
fixed